src/shared/components/Button/Button.tsx

```diff
@@ -8,7 +8,7 @@ import { TextVariant } from '../Text'
 export type ButtonProps = {
   icon?: React.ReactNode
   iconPlacement?: IconPlacement
-  children: string
+  children?: string
```

Comment:
I would still keep the children required and modify props type in `ActionDialog` by omitting `children` like this: 
```tsx
// ActionDialog.tsx
type DialogButtonProps = {
  text: string
  disabled?: boolean
  onClick?: (e: React.MouseEvent) => void
} & Omit<ButtonProps, 'children'>
```
I think it makes sense to do it that way. We don't need to pass children(which is a string in this case) to primary button(or secondary button), because we already have a `text` property

Reply:
Yeah I think this is a good idea